### ISSUE TITLE
need ext prop Extent

### ISSUE THREAD

**Issue by VladimirAlexiev** [american-art/aac_mappings#53]
@azaroth42 CCO has a prop Extent that's used to describe a particular aspect of an object, for the purposes of contribution, material, dimension, or subject.

After analysis of values, I've argued in the CONA confluence that this cannot be mapped to an object Part because many extents are not physical parts.

GM have per-extent dimensions, so lacking proper guidance, they've used P46_is_composed_of improperly (that's for physical parts), eg

http://data.americanartcollaborative.org/GM/object/39895:
```ttl
crm:P46_is_composed_of
  <http://data.americanartcollaborative.org/GM/object/39895/Overall>
```

http://data.americanartcollaborative.org/GM/object/9:	
```ttl
crm:P46_is_composed_of
  <http://data.americanartcollaborative.org/GM/object/9/Framed>
  <http://data.americanartcollaborative.org/GM/object/9/Image>
```

An object cannot be composed of parts `Overall` nor `Framed`.

**Comment by azaroth42** [comment]
Correct, both would be just the object.  Image on the other hand is an acceptable part.
This isn't a problem with the model, but with mapping Overall and Framed to parts rather than the whole... which would be a bug in the mapping implementation for specific institutions.

Propose close.

**Comment by VladimirAlexiev** [comment]
The point is that Extent (as specified by CCO), takes on all kinds of weird and wonderful values in different museums. Most of these do not map to physical parts.

**Comment by VladimirAlexiev** [comment]
@workergnome 
Another example: http://data.americanartcollaborative.org/page/wam/object/33060
crm:P46_is_composed_of wam:object/33060/overall

> bug in the mapping implementation for specific institutions.

IMHO they have followed the letter of the [Dimensions (Part)](http://review.americanartcollaborative.org/entity/E22_Man-Made_Object#field_20-search_0) mapping. linked.art also talks only of Parts: http://linked.art/model/object/physical/#parts.

Museums have thesauri of Extents. Some of them map to parts but others don't (nor are the same as the object itself). That's why I think we need a new prop "extent"

**Comment by workergnome** [comment]
I'm loathe to introduce new properties every time there's something in museum practice that's different than how the CRM models things—is there a pattern for recording this that uses existing CRM constructs, but that can handle extents and parts the same way?

an 'extent' property would have to link to an entity, and I don't know what that entity would be.  I agree it's not really a "part", but it's all not a feature, nor a MMO at all.  It's just a label for a particular form of dimension.

What we're really looking for here is a way to gather a collection of dimensions into a cluster, and say that they go together, because they are related conceptually or through practice.

**Comment by VladimirAlexiev** [comment]
Extents apply not only to dimensions, but also subjects, materials and contributions.
As CCO examples show, some extents are neither parts nor features (eg pattern, repeat, chain lines). More examples eg in https://share.getty.edu/display/ITSLODV/CONA+Dimensions.

If we're collapsing MMO and Feature to MMThing, I don't know what's best to do about Extents. On one hand a rug does not consist of patterns, on the other hand to add a new class while taking away 2 others doesn't seem "right"..

**Comment by VladimirAlexiev** [comment]
CONA has the following values (CONAListsAATLinks_master.xlsx, coreferenced to AAT concepts **and terms**). 
I made a merged list from MatDimExtent (material & dimensions), PersonCorpExtent (contribution), DepictedSubRelsExtent (subject):
- addition, ancillary work, base, binding, body, bowl, boxes, building, carving, cast, ceiling, chain lines, choir, common, component, container, cover, decoration, design, dome, embroidery, engraving, execution, exterior, figures, folio (leaf), foot, footprint, frame, furnishings, grounds, group, handles, illumination, image, interior, item, items, justification, knots, laid lines, lid, lot, mat, mount, mouth, nave, neck, negative, not applicable, object, overall, page, panel, part, pattern, plate (illustration), platemark, portfolio, predella, previous design, primary, primary work, print, repeat, restoration, retouching, revision, rim, roof, rooms, sculpture, secondary, secondary support, sheet, shoulder, site, spire, stand, stories, subgroup, support, surface, tessera, text block, top, tower, transept, undetermined, vessel, volume, work depicted
- <positional extent>: ancillary work, background, bottom, center, component, corner, east, end, exterior, foreground, front, interior, left, lower left, lower right, main work, north, northeast, northwest, obverse, overall, rear, recto, reverse, right, side, side A, side B, south, southeast, southwest, top, upper left, upper right, verso, west, ,
- <non-positional attributes>: dedication, design element, language, method of representation, purpose, related event, 

This should inform the decision whether we can treat these as parts. 
Note that some of them are activities (eg "previous design")

**Comment by workergnome** [comment]
These all appear to be ways to talk about conceptual subparts or regions of a thing.  Perhaps we're making a distinction between parts-which-are-objects, and parts-which-are-conceptual here?

**Comment by VladimirAlexiev** [comment]
Not all are conceptual subparts. Some are "periodic features" (eg chain-lines, laid lines), others are activity parts (execution, previous design, design elements) or related activities (related event)

**Comment by workergnome** [comment]
At that point, Extent is really just a free-text field with a vague conceptual or semantic understanding of what it means.  It doesn't _mean_ anything, it's just a LinguisticObject with type aat:extent.

**Comment by VladimirAlexiev** [comment]
1. It's used with contribution, material, dimension, or subject. So you need to use it as an attribute on a relation.
2. It's a list of AAT concepts just like object type, title type etc.

I don't see how LinguisticObject models any of the above

**Comment by workergnome** [comment]
Any list that allows all of the values "not applicable", "language", and "top" says to me that it's not a concrete semantic distinction.  It's a catchall place to hold human-language expressions that would be valid when discussing the human-language concepts held by the word "extent", which are varied.
